src/Components/Articles/ArticleContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { ArticlesList } from "./ArticlesList";
+import { getArticles } from "../../api";
+
+export const ArticleContainer = () => {
+  const [articles, setArticles] = useState([]);
+  const [searchTerm, setSearchTerm] = useState("");
+
```

Comment:
It would be good to get a loading pattern in here, on a slow connection I have no idea if your site is working as intended or not.